src/gui_common/MainMenu.tscn

```diff
@@ -324,6 +324,17 @@ __meta__ = {
 
 [node name="GUIAnimations" type="AnimationPlayer" parent="."]
 anims/MenuSlide = SubResource( 1 )
+
+[node name="GLES2 Popup" type="ConfirmationDialog" parent="."]
+margin_right = 515.0
+margin_bottom = 108.0
+theme = ExtResource( 2 )
+window_title = "GLES2 Mode Warning"
+dialog_text = "You are running Thrive with GLES2. This is severely untested and is likely to cause issues. Try to update your video drivers or force AMD/Nvidia graphics to be used."
```

Comment:
`Try to update your video drivers and/or force AMD or Nvidia graphics to be used to run Thrive.`
I think that would be a bit better form to say basically the same thing.